src/main/scala/nl.knaw.dans.easy.properties/app/repository/sql/package.scala

```diff
@@ -28,14 +28,31 @@ package object sql {
 
   val dateTimeFormatter: DateTimeFormatter = ISODateTimeFormat.dateTime()
   val timeZone: DateTimeZone = DateTimeZone.UTC
+
   implicit def timeZoneToCalendar(timeZone: DateTimeZone): Calendar = {
     Calendar.getInstance(timeZone.toTimeZone)
   }
+
   implicit def dateTimeToTimestamp(dt: DateTime): Timestamp = new Timestamp(dt.getMillis)
 
   type PrepStatementResolver = (PreparedStatement, Int) => Unit
+
   def setString(s: String): PrepStatementResolver = (ps, i) => ps.setString(i, s)
+
   def setDepositId(depositId: DepositId): PrepStatementResolver = setString(depositId.toString)
+
   def setInt(int: => Int): PrepStatementResolver = (ps, i) => ps.setInt(i, int)
+
   def setInt(s: String): PrepStatementResolver = setInt(s.toInt)
+
+  implicit class RichPreparedStatement(val preparedStatement: PreparedStatement) extends AnyVal {
```

Comment:
Can this be moved to `CommonResultSetParsers`?

Reply:
If possible, that class would also need another name as it is not about the result set but about input parameters for the prepared query.

Reply:
Something more broke tests https://github.com/DANS-KNAW-jp/easy-deposit-properties/pull/2